mantle/platform/qemu.go

```diff
@@ -550,6 +555,21 @@ func (builder *QemuBuilder) setupNetworking() error {
 	return nil
 }
 
+func (builder *QemuBuilder) setupSecondaryNetworking() error {
+	macCounter := 0
+	for i := 1; i <= builder.secondaryNics; i++ {
+		netSuffix := fmt.Sprintf("%d", i)
+		macSuffix := fmt.Sprintf("%02x", macCounter)
+
+		netdev := fmt.Sprintf("tap,id=secondarynet%s,ifname=tap%s,script=no,downscript=no", netSuffix, netSuffix)
```

Comment:
Do we actually need these additional NICs to be TAP devices? AFAIK, those require running QEMU as root whereas focus in kola has been on running qemu unprivileged (see discussions in https://github.com/coreos/fedora-coreos-tracker/issues/340).

Seems like we could just add more user-mode NICs instead here, no?

Reply:
This seems to work:

```
$ kola qemuexec -U -- kola qemuexec -U -- -netdev user,id=eth1 -device virtio-net-pci,netdev=eth1 -netdev user,id=eth2 -device virtio-net-pci,netdev=eth2
...
sh-5.0# ip addr
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host
       valid_lft forever preferred_lft forever
2: ens4: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000
    link/ether 52:54:00:12:34:56 brd ff:ff:ff:ff:ff:ff
    altname enp0s4
3: ens5: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000
    link/ether 52:54:00:12:34:57 brd ff:ff:ff:ff:ff:ff
    altname enp0s5
4: ens7: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000
    link/ether 52:54:00:12:34:58 brd ff:ff:ff:ff:ff:ff
    altname enp0s7
```

Reply:
Hey @jlebon , thanks for your reply!
Let me check if that ticks all the marks that I need, I'll update here soon.